scripts_src/den/dcfranki.ssl

```diff
@@ -100,6 +100,19 @@ procedure Node984;
 #define home_tile                         (local_var(LVAR_Home_Tile))
 #define sex_time_up                       ((game_time > local_var(LVAR_Next_Sex_Time)) and (local_var(LVAR_Next_Sex_Time) != 0))
 
```

Comment:
It's basically a new version of the start_dialog_at_node but with the specific TH and background needed for the character. I should've made it a macro in a header file you're right, just thought this may be easier to sort.